geospace-lib/src/main/scala/com/socrata/geospace/lib/regioncache/SpatialIndex.scala

```diff
@@ -113,7 +111,7 @@ object SpatialIndex {
   /**
    * Create a SpatialIndex[String] from a Layer/FeatureSource.  The feature ID will be stored in the index.
    *
-   * @param layer an [[org.geoscript.layer.Layer]] or GeoTools FeatureSource.
+   * @param layer an org.geoscript.layer.Layer or GeoTools FeatureSource.
```

Comment:
Oops, I did this when findJars was failing to unblock myself and then forgot to revert. See latest commit :)